src/include/common/typedefs.h

```diff
@@ -211,12 +211,14 @@ namespace terrier {
 using byte = std::byte;
 }
 
-STRONG_TYPEDEF(timestamp_t, uint64_t);
-STRONG_TYPEDEF(layout_version_t, uint32_t);
+STRONG_TYPEDEF(boolean_t, int8_t);
 // TODO(Matt): consider namespacing strong typedefs so we have storage::col_id_t and catalog::col_oid_t
 STRONG_TYPEDEF(col_id_t, uint16_t);
 STRONG_TYPEDEF(col_oid_t, uint32_t);
+STRONG_TYPEDEF(date_t, int32_t);
```

Comment:
Signed seems odd to me, but there may be a good reason for this.

Reply:
Looking at the Peloton code, it has an inconsistent approach to date and time stamp values, with respect to signed / unsigned. I think we fix it now and have them both be unsigned.